Formula/vapoursynth-descale.rb

```diff
@@ -0,0 +1,34 @@
+class VapoursynthDescale < Formula
+  desc "VapourSynth plugin to undo upscaling"
+  homepage "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale"
+  url "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale/archive/refs/tags/r6.tar.gz"
+  sha256 "af68c424c146f429e72401e5095f78b57c6a9324a1d19600a9cf0fa85c6bf9c3"
+  license "WTFPL"
+  head "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale"
```

Comment:
`head` link should, by convention in Homebrew Core, end in `.git`, so `https://github.com/<user>/<repo>.git`.